aws_lambda_builders/workflows/nodejs_npm_esbuild/esbuild.py

```diff
@@ -296,3 +297,14 @@ def _get_explicit_file_type(self, entry_point, entry_path):
                 return entry_point + ext
 
         raise ActionFailedError("entry point {} does not exist".format(entry_path))
+
+    @staticmethod
+    def _get_dash_delimited_argument(arg: str) -> str:
```

Comment:
nit: could `_convert_snake_case_to_kebab_case` be a better function name?

Reply:
`_convert_snake_case_to_kebab_case` is a few characters too long for our linter's liking. Changed it to `_convert_snake_to_kebab_case`.